var/spack/repos/builtin/packages/lz4/package.py

```diff
@@ -31,13 +31,21 @@ class Lz4(Package):
     an extremely fast decoder, with speed in multiple GB/s per core,
     typically reaching RAM speed limits on multi-core systems."""
 
-    homepage = "http://cyan4973.github.io/lz4/"
-    url      = "https://github.com/Cyan4973/lz4/archive/r131.tar.gz"
```

Comment:
You might want to still keep a URL for the latest version. `spack fetch` may work without it, but `spack versions` and `spack checksum` won't.